interactions/api/models/attrs_utils.py

```diff
@@ -157,6 +157,15 @@ def inner_convert_dict(dict):
     return inner_convert_dict
 
 
+def convert_type(type_: type):
```

Comment:
you don't use this anywhere?

Reply:
See line L188 - admittedly, it's only really used behind the scenes right now, but it could be expanded into being used elsewhere in the future.

Reply:
Ah thanks, I didnt see it before